oak_utils/src/lib.rs

```diff
@@ -171,30 +190,25 @@ pub fn compile_protos<P>(inputs: &[P], includes: &[P])
 where
     P: AsRef<std::path::Path>,
 {
-    compile_protos_with(true, inputs, includes);
+    compile_protos_with_options(inputs, includes, ProtoOptions::default());
 }
 
-/// Build Rust code corresponding to a set of protocol buffer messages, skipping any service
-/// definitions, emitting generated code to crate's `OUT_DIR`.
-pub fn compile_protos_without_services<P>(inputs: &[P], includes: &[P])
+/// Like `compile_protos`, but allows for configuring options through
+/// [`ProtoOptions`](struct.ProtoOptions.html).
```

Comment:
Is the URL (`struct.ProtoOptions.html`) really needed?

Reply:
Not really I suppose, I'll simplify it to just`ProtoOptions`